docs/supported_docker_environment/index.md

```diff
@@ -2,16 +2,17 @@
-| Mac OS X - Docker for Mac      | 1.12.0          | *Support is best-efforts at present*. `getTestHostIpAddress()` is [not currently supported](https://github.com/testcontainers/testcontainers-java/issues/166) due to limitations in Docker for Mac. |
-| Windows - Docker Toolbox            |                             | *Support is limited at present and this is not currently tested on a regular basis*.                                                                                                                                                                                                                                                                               |
-| Windows - Docker for Windows   |                             | *Support is best-efforts at present.* Only Linux Containers (LCOW) are supported at the moment. See [Windows Support](windows.md)                                                                                                                                                                                                                                                                                                                                        |
+| Host Operating System / Environment | Minimum recommended docker versions | Known issues / tips |
+|----------------------------------------|-----------------------------|--------------------------|
+| Linux - general                        | Docker v1.10                | |
+| Linux - Travis CI                      | Docker v1.10                | See [example .travis.yml](https://raw.githubusercontent.com/testcontainers/testcontainers-java/master/.travis.yml) for baseline Travis CI configuration. |
+| Linux - Circle CI (LXC driver)         | Docker v1.9.1               | The `exec` feature is not compatible with Circle CI. See [example circle.yml](https://raw.githubusercontent.com/testcontainers/testcontainers-java/master/circle.yml) for baseline CircleCI configuration. |
+| Linux - within a Docker container      | Docker v1.12                | See [Running inside Docker](continuous_integration/dind_patterns.md) for Docker-in-Docker and Docker wormhole patterns. |
+| Mac OS X - Docker Toolbox              | Docker Machine v0.8.0       | |
+| Mac OS X - Docker for Mac              | 1.12.0                      | *Support is best-efforts at present*. `getTestHostIpAddress()` is [not currently supported](https://github.com/testcontainers/testcontainers-java/issues/166) due to limitations in Docker for Mac. |
+| Windows - Docker Toolbox               |                             | *Support is limited at present and this is not currently tested on a regular basis*. |
+| Windows - Docker for Windows           | Docker v17.06               | *Support is best-efforts at present.* Only Linux Containers (LCOW) are supported at the moment. See [Windows Support](windows.md). |
```

Comment:
Just wondering, why is there a diff in all those other lines?

Reply:
All these lines had an enormous amount of whitespace characters to fill the last column to make the last pipe _(`|`)_ align.
They took up to three lines of my console on a `1920 x 1080` monitor.
IMHO, that was simply unnecessary and instead of making the code look more ordered _(which cab be said for the first two columns)_ it made it worse.